src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -97,10 +97,12 @@ private constructor(
         val conn = startAgentProcess()
         val client = CodyAgentClient()
         client.onSetConfigFeatures = project.service<CurrentConfigFeatures>()
+        logger.warn("Starting json-rpc Launcher")
```

Comment:
Shouldn't all that logs be on the `info` level?

Reply:
I added a few extra logging statements at key lifecycle stages of starting the Agent, because I found it tremendously helpful in narrowing down where it was failing when the Agent is pooping out. Which is often. I'll review them all, though, to see which ones we should really keep.